.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
-  MINIMUM_NOIR_VERSION: v1.0.0-beta.3
+  MINIMUM_NOIR_VERSION: v1.0.0-beta.4
```

Comment:
@kashbrti should we update this repo's README: https://github.com/noir-lang/noir-bignum/blob/873d901dbb2ae3bc461928f914720f5c06c779a4/README.md?plain=1#L49